### ISSUE TITLE
iOS - Opening and Closing the Notifications center on iOS dismisses the New line Marker

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#20974]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open app
2. Go to any previous chat with reply
3. Marks as unread the reply
4. Open and close the Notification Center
5. Notice the new line marker disappear

## Expected Result:
Opening and closing the Notification Center should not have dismiss the new line marker.

## Actual Result:
Opening and closing the Notification Center dismiss the new line marker

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.28.7

**Reproducible in staging?:** yes

**Reproducible in production?:** yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/be2a9fff-ae05-4555-a569-7058f84a90a0

https://github.com/Expensify/App/assets/93399543/dd44ac3e-a206-4ce1-ab6e-50b615aaffe3

**Expensify/Expensify Issue URL:**

**Issue reported by:** @daveSeife

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1686417083178409

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by s-alves10** [comment]
I'm not sure if this is a bug.
We're currently changing unread status of the report in visibility change event handler when the chat is fully visible.

https://github.com/Expensify/App/blob/cd4c24609fa736e7ddb7cea5ce6938908c9ab601/src/pages/home/ReportScreen.js#L140-L148

If this is an unexpected behavior, we need to define how to change the unread status for the report.

**Comment by abekkala** [comment]
I'm not able to reproduce this. 
When I view the video it looks like the "new" message appears for a moment but since you're on the chat it's then showing "read"

When I tested I kept the chat open and repeated your steps - it would be expected that once I reopen to the same chat that it would momentarily show "new" but since I'm viewing it, it moves to read. 

When I test but after marking as unread, I move back to my LHN, Close and reopen notification, the message remains new/unread as I'm not actually viewing the conversation yet.
